README.md

```diff
@@ -83,17 +83,14 @@ Self-Aware Anomaly Detection System. arXiv preprint arXiv:2011.05047.* ([arxiv l
 
 ## Other Useful Resources
 
-1. *Chakraborty, S., Shah, S., Soltani, K., & Swigart, A. (2019, December). Root Cause Detection Among Anomalous Time 
+- *Chakraborty, S., Shah, S., Soltani, K., & Swigart, A. (2019, December). Root Cause Detection Among Anomalous Time 
 Series Using Temporal State Alignment. In 2019 18th IEEE International Conference On Machine Learning And Applications 
 (ICMLA) (pp. 523-528). IEEE.* ([arxiv link](https://arxiv.org/abs/2001.01056))
 
 
-## Acknowledgements
```

Comment:
Why are we removing this part?

Reply:
citations are already added in the actual paper and we already linked the paper in the README